twopaneview/examples/metro.config.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Metro configuration for React Native
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ */
+
+const blacklist = require('metro-config/src/defaults/blacklist');
+
+module.exports = {
+  resolver: {
+    blacklistRE: blacklist([
+      /node_modules\/.*\/node_modules\/react-native\/.*/,
+    ])
+  },
+  transformer: {
+    getTransformOptions: async () => ({
+      transform: {
+        experimentalImportSupport: false,
+        inlineRequires: false,
+      },
+    }),
+  },
+};
```

Comment:
I think we can remove this file for now.  When we add support for windows, we'll need to do something similar to:  [https://github.com/react-native-community/react-native-webview/pull/1220](https://github.com/react-native-community/react-native-webview/pull/1220)

Reply:
Removed